### PR TITLE
Override stride-related methods in LTCTensorImpl

### PR DESCRIPTION
**Pull request by desertfire**
Summary: Strides on lazy tensor should only be read after calling setup_size_properties. Fixes a failure in hf_Longformer.

### CODE REVIEW

**Review by alanwaketan — COMMENTED** [review]
Thanks, Bin. I just landed LTCTensorImpl to master #70062. Wondering if you mind holding this after we merge master back and reapply this change to the torch/csrc/lazy version.

Do you think we can come out with a test case for this change?

**Comment by desertfire** [comment]
> 

Since it only affects hf_Longformer so far, we can hold this until you finish the merging. I will add a simple python test.

**Comment by alanwaketan** [comment]
FYI, here is the PR to keep track of the latest merging master into lazy_tensor_staging.

**Comment by alanwaketan** [comment]
Should we update this to reflect your latest change in #70623?

**Review by wconstab — APPROVED** [review]
this LGTM, except i was a little confused about why the change is made twice.  i guess the lazy_tensor_core/csrc copy of this file needs to be updated separately (this PR) since it is still being used.  Once the cleanup is done and we switch to the torch/csrc/lazy version of this file, then this goes away right?

**Comment by desertfire** [comment]
Fixed in master with https://github.com/pytorch/pytorch/pull/70623. Close this one.

**Pull request closed without merging**
